.github/workflows/build.yml

```diff
@@ -28,15 +28,21 @@ jobs:
 
     steps:
     - name: Checkout sources
-      uses: actions/checkout@v2
+      uses: actions/checkout@v4
+
+    - name: Mark repository as safe
+      run: git config --global --add safe.directory "$(pwd)"
+
+    - name: Install Build Requirements
+      run: dnf install -y $(grep BuildRequires ioprocess.spec.in |cut -f2 -d:)
```

Comment:
dnf builddep --spec ioprocess.spec.in

Isn't that easier? :)

Reply:
Would be easier but it fails with:

```
RPM: error: line 6: Illegal char '@' (0x40) in: Version:	@VERSION@
Failed to open: 'ioprocess.spec.in', not a valid spec file: can't parse specfile
```